ui/src/app/stores/workflows/workflows.effects.ts

```diff
@@ -298,13 +314,28 @@ export class WorkflowsEffects {
             },
           ];
         }),
-        catchError(() => {
-          this.toastrService.error(texts.UPDATE_WORKFLOW_FAILURE_NOTIFICATION);
-          return [
-            {
-              type: WorkflowActions.UPDATE_WORKFLOW_FAILURE,
-            },
-          ];
+        catchError((errorResponse) => {
+          if (
+            errorResponse instanceof Array &&
```

Comment:
You may want to extract this if-condition

Reply:
Fixed